composables/getRemainingDays.test.ts

```diff
@@ -0,0 +1,33 @@
+import { getRemainingDaysInMonth } from './getRemainingDays'
+
+describe('getRemainingDaysInMonth', () => {
```

Comment:
I like the idea of having the these specific tests for each piece of the countdown